packages/caver-utils/src/index.js

```diff
@@ -505,6 +505,8 @@ module.exports = {
     padRight: utils.rightPad,
     rightPad: utils.rightPad,
     toTwosComplement: utils.toTwosComplement,
+    isTransaction: utils.isTransaction,
```

Comment:
How about changing name from isTransaction to isTransactionHash or isTxHash?

Reply:
Sounds good. I'll update the names.